data/.schema.yaml

```diff
@@ -33,14 +33,43 @@ $defs:
         description: Specimen type.
         type: string
         enum:
-        - stool
-        - sputum
-        - throat_swab
+          - stool
+          - sputum
+          - throat_swab
+      biomarker:
+        description: Biomarker being measured.
+        type: string
+        enum:
+          - SARS-CoV-2
+          - mtDNA
+          - PMMoV
```

Comment:
Can we add crAssphage to this enum list based on its inclusion in the Arts et al. (2023) paper? Sorry for the delayed notice.

Reply:
No problem! While you're adding the data from Arts et al. (2023), you can simply add `crAssphage` to the list on your local machine and include the changes the next time you commit. Let me know if that doesn't work, and we can find another way.

Reply:
I see! I didn't know I could directly change the schema. Thanks for doing it for me this time. I'll make sure to handle it next time.

Reply:
This PR is already merged so we won't be able to make further changes here. Probably the easiest approach is for you to add it. If that doesn't work for some reason, I can create another PR.

Reply:
No problem! I can do it! Thanks for letting me know.